configure

```diff
@@ -182,15 +203,28 @@ gen_ssl()
 		msg skip "  skipping $PREFIX/secrets/ssl.yaml; already exists."
 		return
 	fi
-
+	echo "Generating SSL certificates... This can take up to a minute."
 	# generate new certificates
-	OUTPUT=$(certbot -d "$HUB_FQDN" \
-		--keep --manual \
-		--preferred-challenges dns certonly \
-		--config-dir $PREFIX/secrets/certbot/config --work-dir $PREFIX/secrets/certbot/work --logs-dir $PREFIX/secrets/certbot/logs \
-		-m "$LETSENCRYPT_EMAIL" \
-		--agree-tos --no-eff-email --manual-public-ip-logging-ok \
-		--manual-auth-hook=bin/certbot-auth.sh 2>&1)
+	if [[ $PROVIDER == do ]]; then
```

Comment:
Since these two scripts are called with exactly the same arguments, I'd propose replacing the if block with something like:
```
OUTPUT=$(.....
    ....
    --manual-auth-hook=bin/certbot-auth.sh.${PROVIDER}
)
```
(and renaming `certbot-auth.sh` -> `certbot-auth.sh.do`).

Reply:
agreed, made these changes.